packages/babel-cli/src/babel/dir.js

```diff
@@ -71,7 +71,14 @@ export default function(commander, filenames, opts) {
       if (commander.deleteDirOnStart) {
         util.deleteDir(commander.outDir);
       }
-      util.readdir(dirname).forEach(function(filename) {
+
+      let readDirFilter;
+
+      if (commander.copyDotFiles) {
+        readDirFilter = () => true;
```

Comment:
@existentialism Does this also ignore `.` and `..`? Would `(x) => x !== '.' && x !== '..'` work better? I'm sure you've done the tests and all that, so I'm just curious. 👍